source/digits_hits/src/GateProtonNuclearInformationActor.cc

```diff
@@ -0,0 +1,106 @@
+/*----------------------
+  Copyright (C): OpenGATE Collaboration
+
+  This software is distributed under the terms
+  of the GNU Lesser General  Public Licence (LGPL)
+  See GATE/LICENSE.txt for further details
+  ----------------------*/
+
+#include "GateConfiguration.h"
+
+// Gate 
+#include "GateProtonNuclearInformationActor.hh"
+#include "GateMiscFunctions.hh"
+
+// G4
+#include <G4Event.hh>
+#include <G4ParticleTable.hh>
+#include <G4TransportationManager.hh>
+#include "G4HadronicProcessStore.hh"
+#include "G4HadronicProcess.hh"
+
+#define TRY_AND_EXIT_ON_ITK_EXCEPTION(execFunc)                         \

```

Comment:
You define a macro here, but I do not see where you are actually using it. Am I missing something, or is this maybe a leftover from a previous version, and these lines could be removed?

Reply:
Leftover it seems. Sorry for not taking the time to fully clean up Cath's code. I have removed this macro. Thanks!